Sources/Command/Run/CommandContext.swift

```diff
@@ -12,6 +12,9 @@ public struct CommandContext {
 
     /// The parsed options (according to declared signature).
     public var options: [String: String]
+    
+    /// Extraneous arguments that did not match the commands defined arguments.
```

Comment:
maybe mark here a note about strictness? it's fine either way since it's pretty obvious